finalPrep/README.md

```diff
@@ -18,37 +18,43 @@ Some things to consider as you refactor include...
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here`<img width="1119" alt="Screen Shot 2021-09-10 at 7 22 56 PM" src="https://user-images.githubusercontent.com/89324625/132931807-5107c530-4747-4555-9e08-ffaac0867145.png">
+<img width="1123" alt="Screen Shot 2021-09-10 at 7 22 38 PM" src="https://user-images.githubusercontent.com/89324625/132931810-70f2fda6-6657-4577-956e-c8171980cd56.png">
+
+- `Add Week 2 Screenshot Here` <img width="1146" alt="Screen Shot 2021-09-10 at 7 23 22 PM" src="https://user-images.githubusercontent.com/89324625/132931803-9f9a3852-8703-4dad-90cf-58ffb8f776bb.png">
+<img width="1159" alt="Screen Shot 2021-09-10 at 7 23 10 PM" src="https://user-images.githubusercontent.com/89324625/132931806-a08dc7ef-88f8-423f-961e-0cbbeac52809.png">
+
+
+- `Add Week 3 Screenshot Here`<img width="1156" alt="Screen Shot 2021-09-10 at 7 23 43 PM" src="https://user-images.githubusercontent.com/89324625/132931800-7701a4d5-49a4-4762-b92f-6af939e85c8e.png">
+<img width="1157" alt="Screen Shot 2021-09-10 at 7 23 34 PM" src="https://user-images.githubusercontent.com/89324625/132931802-10eff464-b1a9-42bf-893d-2d700d57f013.png">
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist [here:](https://gist.github.com/StevieAmb/11f4908fbc4639e35a45f96413698459)
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist [here:](https://gist.github.com/StevieAmb/0e09bee573406d01729d6c633e03e925)
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+  - [ ] Add a link to your gist [here:](https://gist.github.com/StevieAmb/cfcbfb1588834e402967908110ac328e)
```

Comment:
It sounds like you'll bring a lot of empathy, and an emphasis on clear, honest communication to a team. I love the bit about being open to learn how your teammates best communicate and learn best!